lib/linkedin-scraper/profile.rb

```diff
@@ -91,10 +91,12 @@ def current_companies
     def education
       @education ||= @page.search('.background-education .education').map do |item|
         name   = item.at('h4').text.gsub(/\s+|\n/, ' ').strip      if item.at('h4')
-        desc   = item.at('h5').text.gsub(/\s+|\n/, ' ').strip      if item.at('h5')
-        period = item.at('.education-date').text.gsub(/\s+|\n/, ' ').strip if item.at('.education-date')
-
-        {:name => name, :description => desc, :period => period }
+        desc   = item.search('h5').last.text.gsub(/\s+|\n/, ' ').strip      if item.search('h5').last
+        degree = item.search('h5').last.at('.degree').text.gsub(/\s+|\n/, ' ').strip.gsub(/,$/, '')      if item.search('h5').last.at('.degree')
+        major   = item.search('h5').last.at('.major').text.gsub(/\s+|\n/, ' ').strip      if item.search('h5').last.at('.major')
```

Comment:
Line is too long. [128/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.